libs/blocks/modal/modal.js

```diff
@@ -176,7 +174,7 @@ export async function getModal(details, custom) {
       .forEach((element) => element.setAttribute('aria-disabled', 'true'));
   }
   if (dialog.classList.contains('commerce-frame')) {
```

Comment:
I'm also not sure if the commerce-frame actually belongs into the generic modal code

Reply:
It was done as part of another ticket to fix modal size in mobile and tablet if it is opening a commerce iframe in the modal. PR https://github.com/adobecom/milo/pull/1031

Reply:
commerce iframe is using the same modal to open and close just applies a specific css class to define its height and width.

Reply:
PR #1031 should have had the logic tied into commerce as well. The modal code is loaded in a lot of use-cases, commerce being a very specific one.

IMO very specific code that solves for _one_ use case shouldn't belong into milo core - but the originating logic, in this case commerce.

I'd love to see someone from the @adobecom/admins weigh in on this